Purchases/Public/Purchases.swift

```diff
@@ -751,23 +756,33 @@ public extension Purchases {
      * Get latest available customer  info.
      *
      * - Parameter completion: A completion block called when customer info is available and not stale.
-     * Called immediately if ``CustomerInfo`` is cached. Customer info can be nil * if an error occurred.
+     * Called immediately if ``CustomerInfo`` is cached. Customer info can be nil if an error occurred.
      */
     @objc func getCustomerInfo(completion: @escaping (CustomerInfo?, Error?) -> Void) {
         customerInfoManager.customerInfo(appUserID: appUserID, completion: completion)
     }
 
     /**
      * Get latest available customer  info.
+     * Returns a value immediately if ``CustomerInfo`` is cached.
      *
-     * - Parameter completion: A completion block called when customer info is available and not stale.
-     * Called immediately if ``CustomerInfo`` is cached. Customer info can be nil * if an error occurred.
+     * - Seealso `Purchases.customerInfoStream`
      */
     @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
     func customerInfo() async throws -> CustomerInfo {
         return try await customerInfoAsync()
     }
 
+    /// Returns an ``AsyncStream`` of ``CustomerInfo`` changes.
+    ///
+    /// - Seealso `PurchasesDelegate.purchases(_ purchases: Purchases, didReceiveUpdated:)`
+    /// - Seealso `Purchases.customerInfo()`
+    /// - Note: this method is not thread-safe.
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
```

Comment:
we don't support watch 6.0, min deployment target on watch is 6.2 since previous versions didn't even have IAP

Reply:
Weird, I copied this from other places, and I see there's 28 that use:
```
@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.0, *)
```

I'll send another PR to correct them.

Reply:
https://github.com/RevenueCat/purchases-ios/pull/1143

Reply:
oops, great catch!